src/main/resources/services/dashboard/dashboard.es6

```diff
@@ -33,17 +33,18 @@ exports.get = function(req) {
   if (!req.params || !req.params.id) {
     return missingParameterResponse()
   }
+
   logDataQueryEvent(req.params.id, {
-    message: Events.GET_DATA_STARTED
-  })
-  const updateResult = context.run(createContextOption('master'), () => {
-    return getAllOrOneDataQuery(req.params.id).map((dataquery) => updateDataQuery(dataquery) )
+    message: Events.STARTED
   })
 
+  const updateResult = getAllOrOneDataQuery(req.params.id).map((dataquery) => updateDataQuery(dataquery))
+
   const parsedResult = updateResult.map(fontEndObject)
```

Comment:
Suggest something like `transfromQueryResult` instead of `frontEndObject`

Reply:
changing name now